.github/workflows/release-github.yml

```diff
@@ -0,0 +1,21 @@
+name: Github Release
+
+on:
+  push:
+    tags:
+      - 'v*.*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Release
+        uses: softprops/action-gh-release@v1
+        if: startsWith(github.ref, 'refs/tags/')
+        with:
+          files: |
+            target/*.jar
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
this needs to be adapted to something like this in order to work https://github.com/nevermined-io/sdk-py/blob/master/.github/workflows/release-github.yml